doc/manual/source/introduction.md

```diff
@@ -1,8 +1,8 @@
 # Introduction
 
 Nix is a _purely functional package manager_.  This means that it
-treats packages like values in purely functional programming languages
-such as Haskell — they are built by functions that don’t have
+treats packages like values in a purely functional programming language
```

Comment:
```suggestion
treats packages like values in a purely functional programming language (such as Haskell)
```
I'm a bit sentimental! :)

Reply:
With parens, would it really confuse people? Or would it be clear it's an optional reference --- helpful if you know it, fine if you don't know it.

Reply:
The thing is, Nix has nothing to do with Haskell, but among bystanders (at least last time I checked) had a similar aura of elitist mystery. So for the most sensitive audience of perfectly suitable potential users who may benefit from a bit of FP (but because of material results and not because they care what it does to your thinking) it's confusing on two levels: mystery squared, and a priming that there may be some essential relationship.

It's something completely different to say something in an FAQ about the subtle ties between FP projects and communities, but this here is a very different place with different expectations from and different effects on readers.

And priming is powerful.